src/components/Areas/Areas.js

```diff
@@ -5,7 +5,7 @@ import { Link } from 'react-router-dom';
 const Areas = props => {
   const areaCards = props.areaSpecs.map(area => {
     return (
-      <Link className="area-card">
+      <Link to={`/areas/${area.id}`} className="area-card" onClick={() => props.setAreaId(area.id)}>
```

Comment:
This is your call, but destructuring would help clean this up a little bit here, at least for your props. You can write the component:

```javascript
const Areas = ({ areaSpecs, setAreaId }) => {
  ...
}
```

You could technically destructure the `area` object in your `.map` as well:

```javascript
areasSpecs.map(({ id, name, location, about }) => { ... }
```

I personally prefer destructuring, but there's a time and place for it. If you find yourself with a lot of `props.x` or `area.x` it can help to make things a little more explicit/readable.